### ISSUE TITLE
[SF][A11y][Popover]: border color fails to meet 3:1 against its inner white background in sap_horizon theme

### ISSUE THREAD

**Issue by GongRichard** [SAP/ui5-webcomponents#5496]
**Describe the bug**
border color fails to meet 3:1 against its inner white background in sap_horizon theme

**To Reproduce**
1. Go to 'https://sap.github.io/ui5-webcomponents-react/?path=/story/modals-popovers-actionsheet--default-story&globals=theme:sap_horizon'
2. Click on 'Open ActionSheet'
3. Use CCA tool to compare the color

**Expected behavior**
Popovers - The border has to meet 3:1 CC
https://wiki.wdf.sap.corp/wiki/display/Acc/Color+Contrasts+for+Non-Textual+Information#ColorContrastsforNonTextualInformation-Popover,Popup

**Screenshots**
<img width="1386" alt="image" src="https://user-images.githubusercontent.com/13098801/178265283-0a5ee59e-6dfb-432c-a2c4-6855865d6cc6.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: latest
`@ui5/webcomponents-react` version: latest
Operating System:
Browser:

**Additional context**

**Comment by gmkv** [comment]
Consulting with design team.

**Comment by gmkv** [comment]
Hello @GongRichard 
The values for shadow levels have been updated.
When a new version of the [theming variables library is released](https://github.com/SAP/theming-base-content), we will upgrade to it to correct the contrast ratios.
Regards,
Georgi

**Comment by GongRichard** [comment]
Hi @gmkv ,
Is this issue has been fixed? I tested latest wc version 1.7.1, this issue still exists.
Thanks,
Richard

**Comment by gmkv** [comment]
Thanks @GongRichard. I've prepared the PR linked above to update the variables.
The new theming variable library includes the updated shadows for the Popover for better contrast.
For example, here is the current shadow:
![image](https://user-images.githubusercontent.com/22086365/199199358-9e6a63e2-f39b-49de-b832-0837afbf26af.png)

And the updated shadow, with better contrast:
![image](https://user-images.githubusercontent.com/22086365/199199737-8505c7d6-8047-49a5-a7bd-4520c5bbbbae.png)

The fix will be available in the next release of ui5-webcomponents.
